### PR TITLE
Deploy istio

### PR DESCRIPTION
**Pull request by ilackarms**
adds support for the `wasme deploy` command for istio

### CODE REVIEW

**Comment by ilackarms** [comment]
i still need to fix the command description for `deploy istio` and `undeploy istio`

File: pkg/cmd/deploy/deploy.go [hunk 1]
```diff
@@ -52,6 +56,9 @@ func makeDeployCommand(opts *options, provider, use, short, long string, minArgs
 		Args:  cobra.MinimumNArgs(minArgs),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			opts.providerType = provider
+			if minArgs == 1 {
```
Comment (yuval-k) [inline pkg/cmd/deploy/deploy.go]: ?

File: pkg/cmd/deploy/deploy.go [hunk 2]
```diff
@@ -84,6 +91,40 @@ Use --labels to use a match Gateway CRs by label.
 	)
 }
 
+func deployIstioCmd(opts *options) *cobra.Command {
+	use := "istio <image> --id=<unique name> [--config=<inline string>] [--root-id=<root id>] [--namespaces <comma separated namespaces>] [--labels <key1=val1,key2=val2>]"
+	short := "Deploy an Envoy WASM Filter to Istio Sidecar Proxies (Envoy)."
+	long := `Deploy an Envoy WASM Filter to Istio Sidecar Proxies (Envoy).
```
Comment (yuval-k) [inline pkg/cmd/deploy/deploy.go]: maybe note that this is only for istio 1.4 ?

**Pull request merged**